### PR TITLE
Add owner key details screen

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1450

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListFragment.kt [hunk 1]
```diff
@@ -90,6 +92,17 @@ class OwnerListFragment : BaseViewBindingFragment<FragmentOwnerListBinding>(), O
     override fun onResume() {
         super.onResume()
         viewModel.loadOwners()
+
+        //FIXME: find better way to pass results in nav graph
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListFragment.kt:96]: This StackOverflow question might have an answer for us. I am not sure which one really works and is better than what we have today: https://stackoverflow.com/questions/50334550/navigation-architecture-component-passing-argument-data-to-the-startdestination

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    fun loadOwnerDetails(address: Solidity.Address) {
+        safeLaunch {
+
+            val ownerName = credentialsRepository.owner(address)!!.name
+
+            val qrCode = runCatching {
+                qrCodeGenerator.generateQrCode(
+                    address.asEthereumAddressChecksumString(),
+                    512,
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsViewModel.kt:35]: Should this be an extracted constant? Does it have a relationship with the `owner_qr_code` views width and height?

File: app/src/main/res/layout/fragment_owner_details.xml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:text="@string/signing_owner_details_key_address"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/owner_address" />
+
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/owner_qr_code"
+            android:layout_width="124dp"
```
Comment (biafra23) [inline app/src/main/res/layout/fragment_owner_details.xml:137]: This looks rather small on my phone. But it is probably the same size as in the share dialog?

Comment (elgatovital) [reply app/src/main/res/layout/fragment_owner_details.xml:137]: yes

Comment (elgatovital) [reply app/src/main/res/layout/fragment_owner_details.xml:137]: yes

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListFragment.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListFragment.kt:96]: I would create separate issue for this refactoring (and maybe investigating other available alternatives) as it touches places not related to this feature.

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsViewModel.kt (hunk 2 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsViewModel.kt:35]: it's size of a bitmap. I would address this in scope of export owner screen, as there we have yet another qr code :)

**Review by biafra23 — APPROVED** [review]

**Pull request merged**